### ISSUE TITLE
generate.php is failing with only warnings

### ISSUE THREAD

**Issue by tleveque23** [justinwalsh/daux.io#249]
v. 0.1.1
I am trying to generate a static version of the doc using the command:
> php generate.php . static/

(to generate the static files inside an existing directory "static")
But it is failing with only warnings:
> Warning: Invalid argument supplied for foreach() in /Users/thierryleveque/Downloads/daux.io-master/libs/daux_helper.php on line 164
Warning: Invalid argument supplied for foreach() in /Users/thierryleveque/Downloads/daux.io-master/libs/daux_helper.php on line 168
Warning: Invalid argument supplied for foreach() in /Users/thierryleveque/Downloads/daux.io-master/libs/daux_helper.php on line 173
Warning: Invalid argument supplied for foreach() in /Users/thierryleveque/Downloads/daux.io-master/libs/daux.php on line 153

I say it is falling, because in my static directory I can only see "img", "js" and "templates" directories. Not html.

Am I doing something wrong?
I am using the latest master version (0.1.1) and PHP 5.5.14 on OsX 10.10.1
>PHP 5.5.14 (cli) (built: Sep  9 2014 19:09:25)
Copyright (c) 1997-2014 The PHP Group
Zend Engine v2.5.0, Copyright (c) 1998-2014 Zend Technologies

Note that it is working fine running it locally with Grunt.

**Comment by PhilipTobias** [comment]
To generate static documents in the following directory, I used this Mac OS terminal command:

cd /Users/philiptobias/Documents/WWW/Daux/daux.io-master/


then I used this terminal command:

php generate.php global.json docs-output


That generated static HTML documentation in the docs-output folder created at the directory's root level. That new output folder is on the same level as the global.json file and the docs folder used for the source documents.

That worked for me. Good luck.  ...pt
